language/move-lang/tests/functional/translated_ir_tests/validator_set/add_many_validators.move

```diff
@@ -0,0 +1,208 @@
+}
+
+// check: EXECUTED
+
+//! block-prologue
+//! proposer: bob
+//! block-time: 2
+
+// check: EXECUTED
+
+//! new-transaction
+//! sender: association
+use 0x0::LibraSystem2;
+// add allice to validator set
+fun main() {
+    0x0::Transaction::assert(LibraSystem2::validator_set_size() == 0, 99);
+    LibraSystem2::add_validator(&{{alice}});
+    0x0::Transaction::assert(LibraSystem2::is_validator(&{{alice}}), 100);
+    0x0::Transaction::assert(!LibraSystem2::is_validator(&{{bob}}), 99);
+    
+}
+
+// check: EXECUTED
+//! block-prologue
+//! proposer: bob
+//! block-time: 3
+
+// check: EXECUTED
+
+//! new-transaction
+//! sender: association
+use 0x0::LibraSystem2;
+// add bob to validator set
+fun main() {
+    LibraSystem2::add_validator(&{{bob}});
+}
+
+// check: EXECUTED
+
+//! block-prologue
+//! proposer: bob
+//! block-time: 4
+
+// check: EXECUTED
+
+//! new-transaction
+//! sender: association
+// add carrol to the validator set
+use 0x0::LibraSystem2;
+fun main() {
+    LibraSystem2::add_validator(&{{carrol}});
+}
+
+// check: EXECUTED
+
+//! block-prologue
+//! proposer: bob
+//! block-time: 5
+
+// check: EXECUTED
+
+//! new-transaction
+//! sender: association
+// add david to the validator set
+use 0x0::LibraSystem2;
+fun main() {
+    0x0::Transaction::assert(!LibraSystem2::is_validator(&{{david}}), 99);
+    LibraSystem2::add_validator(&{{david}});
+    0x0::Transaction::assert(LibraSystem2::is_validator(&{{david}}), 99);
+}
+
+// check: EXECUTED
+
+//! block-prologue
+//! proposer: bob
+//! block-time: 6
+
+// check: EXECUTED
+
+//! new-transaction
+//! sender: association
+// add eve to validator set
+use 0x0::LibraSystem2;
+fun main() {
+    LibraSystem2::add_validator(&{{eve}});
+}
+
+// check: EXECUTED
+//! block-prologue
+//! proposer: bob
+//! block-time: 7
+
+// check: EXECUTED
+
+//! new-transaction
+//! sender: association
+// add fedor to validator set
+use 0x0::LibraSystem2;
+fun main() {
+    LibraSystem2::add_validator(&{{fedor}});
+    0x0::Transaction::assert(LibraSystem2::validator_set_size() == 6, 99);    
+}
+
+// check: EXECUTED
+//! block-prologue
+//! proposer: bob
+//! block-time: 8
+
+// check: EXECUTED
+
+//! new-transaction
+//! sender: association
+// remove alice
+use 0x0::LibraSystem2;
+fun main() {
+    LibraSystem2::remove_validator(&{{alice}});
+    0x0::Transaction::assert(LibraSystem2::validator_set_size() == 5, 99);
+}
+
+// check: EXECUTED
+//! block-prologue
+//! proposer: bob
+//! block-time: 9
+
+// check: EXECUTED
+
+//! new-transaction
+//! sender: association
+// remove fedor
+use 0x0::LibraSystem2;
+fun main() {
+    LibraSystem2::remove_validator(&{{fedor}});
+    0x0::Transaction::assert(LibraSystem2::validator_set_size() == 4, 99);
+}
+
+// check: EXECUTED
+//! block-prologue
+//! proposer: bob
+//! block-time: 10
+
+// check: EXECUTED
+
+//! new-transaction
+//! sender: association
+// make sure nobody else can be deleted
```

Comment:
would be good to state the assumption of why this is correct

also would be good to try to delete someone that doesn't exist